core/src/main/java/tech/pegasys/web3signer/core/service/http/SigningObjectMapperFactory.java

```diff
@@ -26,17 +26,17 @@
 import tech.pegasys.teku.infrastructure.jackson.deserializers.uints.UInt64Deserializer;
 import tech.pegasys.teku.infrastructure.jackson.deserializers.uints.UInt64Serializer;
```

Comment:
@rolfyone what about uints deserializers? Do we need to migrate those as well?

Reply:
the ones in infra are fine, if we end up moving to TypeDefinitions they won't be referenced ultimately...